ci/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM python:3.7-buster
```

Comment:
I just bumped the server to bullseye, we probably should base our ci on that as well.

Reply:
Done